variantlib/models/variant.py

```diff
@@ -67,69 +122,80 @@ class VariantMeta(BaseModel):
         }
     )
 
-    def __hash__(self) -> int:
-        # Variant Metas are unique in namespace & key and ignore the value.
-        # Class is added to the hash to avoid collisions with other dataclasses.
-        return hash((self.__class__, self.namespace, self.key))
+    def __eq__(self, other: object) -> bool:
+        if not isinstance(other, VariantProperty):
+            return NotImplemented
+
+        return (
+            self.namespace == other.namespace
+            and self.feature == other.feature
+            and self.value == other.value
+        )
+
+    @property
+    def property_hexdigest(self) -> str:
```

Comment:
Is this hash actually going to be used anywhere public? If not, then there's really no point in adding extra complexity and abstraction (= making code less obvious), if `hash()` would suffice for internal needs.

Reply:
Let me try to replace with `hash()` - it should indeed work :)

Reply:
Fixed in 3d7d49b514d83f1c8a83734dfbba720b9393d1f5